lms/djangoapps/teams/static/teams/js/views/team_profile_header_actions.js

```diff
@@ -9,7 +9,7 @@
         function (Backbone, _, gettext, TeamUtils, teamProfileHeaderActionsTemplate) {
             return Backbone.View.extend({
 
-                errorMessage: gettext("An error occurred. Try again."),
+                errorMessage: _.escape(gettext("An error occurred. Try again.")),
```

Comment:
@cahrens 
The i18n docs say we should escape all the translated strings. Thats why i escaped this string. Please correct if my understanding is incorrect.